server/src/test/java/org/eclipse/lsp/cobol/usecases/TestIdmsTransferStatement.java

```diff
@@ -51,19 +58,34 @@
   private static final String TRANSFER_ALL =
       "           TRANSFER CONTROL TO 'TSTPROG' NORETURN USING {$WK1};\r\n";
 
+  private static final String TRANSFER_NO_TERM =
+      "           TRANSFER CONTROL TO 'TSTPROG' LINK {DISPLAY|1} 'ERROR'.\r\n";
+
+  private static final String TRANSFER_ON =
+      "           TRANSFER CONTROL TO 'TSTPROG' NORETURN USING {$WK1};\r\n"
+          + "             ON {$ANY-ERROR-STATUS} DISPLAY 'TRANSFER ERROR'.\r\n";
+
   private static Stream<String> textsToTest() {
     return Stream.of(
         BOILERPLATE + TRANSFER,
         BOILERPLATE + TRANSFER_PARMS,
         BOILERPLATE + TRANSFER_PARMS2,
         BOILERPLATE + TRANSFER_USING_PARMS,
-        BOILERPLATE + TRANSFER_ALL);
+        BOILERPLATE + TRANSFER_ALL,
+        BOILERPLATE + TRANSFER_NO_TERM,
+        BOILERPLATE + TRANSFER_ON);
   }
 
   @ParameterizedTest
   @MethodSource("textsToTest")
-  @DisplayName("Parameterized - IDMS read tests")
+  @DisplayName("Parameterized - IDMS transfer tests")
   void test(String text) {
-    UseCaseEngine.runTest(text, ImmutableList.of(), ImmutableMap.of());
+    UseCaseEngine.runTest(
+        text,
+        ImmutableList.of(),
+        ImmutableMap.of(
+            "1",
+            new Diagnostic(
+                null, MESSAGE_1, DiagnosticSeverity.Error, SourceInfoLevels.ERROR.getText())));
```

Comment:
You expect the error only for one test, not for all. Please, extract this test and run it separately.
Right now UseCase Engine allows passing tests if the number of expected errors greater than actual. But we will fix it soon.
You can create a separate method:
```
@Test
void testWithError() {
    UseCaseEngine.runTest(
        BOILERPLATE + TRANSFER_NO_TERM,
        ImmutableList.of(),
        ImmutableMap.of(
            "1",
            new Diagnostic(
                null, MESSAGE_1, DiagnosticSeverity.Error, SourceInfoLevels.ERROR.getText())));
```

Reply:
I have made the requested changes

Reply:
I don't see the changes, did you push them?

Reply:
Sorry about that, TestIdmsTransferStatement.java is updated now.

Reply:
Thanks)